A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -156,16 +159,21 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
 
     onUpdate() {
         let _deltaTime = this.getDeltaTime();
-
         super.onUpdate(_deltaTime);
 
         const selfTestCurrentKnobValue = SimVar.GetSimVarValue(this.isTopScreen ? "LIGHT POTENTIOMETER:92" : "LIGHT POTENTIOMETER:93", "number");
+        // ECAM all button
+        const ecamAllButtonState = SimVar.GetSimVarValue("L:A32NX_ECAM_ALL_Push_IsDown", "Bool");
```

Comment:
**nitpick**: variable name isn't telling me what it is, comment is used to try and make things clear.
```suggestion
        const ecamAllButtonBeingPushed = SimVar.GetSimVarValue("L:A32NX_ECAM_ALL_Push_IsDown", "Bool");
```